### PR TITLE
[JENKINS-31753] Fields are aligned at the bottom

### PR DESCRIPTION
**Pull request by fbonerandi**
https://issues.jenkins-ci.org/browse/JENKINS-31753

### CODE REVIEW

**Comment by daniel-beck** [comment]
For a UI change, before/after screenshots would be nice.

**Comment by fbonerandi** [comment]
Before

before_modif

after

after_modif

**Comment by oleg-nenashev** [comment]
:+1:

File: core/src/main/resources/hudson/model/ParametersAction/index.jelly [hunk 1]
```diff
@@ -35,12 +35,15 @@ THE SOFTWARE.
 		<st:include page="sidepanel.jelly" it="${build}" />
 		<l:main-panel>
 			<h1>${%Build} ${build.displayName}</h1>
-			<l:pane title="${%Parameters}" width="3">
-			<j:forEach var="parameterValue" items="${it.parameters}">
-				<st:include it="${parameterValue}"
-					page="value.jelly" />
-			</j:forEach>
-			</l:pane>
+			<p>${%Parameters} :</p>
```
Comment (jtnord) [inline core/src/main/resources/hudson/model/ParametersAction/index.jelly]: should this not be a heading or something - it seems to get lost now - whereas before it was styled differently

**Comment by fbonerandi** [comment]
The style is same as the run parameter screen.
The run parameter screen :
run_parameter

**Comment by jtnord** [comment]
@fbonerandi I would say that screen is wrong also.
I'm not against this going in as-is - I just wanted to point it out that it is probably not ideal (even if better than it was).

**Comment by fbonerandi** [comment]
@jtnord  what wrong ?
Please, could you add more information ?

**Comment by jtnord** [comment]
@fbonerandi "This build used the following parameters"  and "Parameters" are headings for the table that follows - yet both are just rendered as simple paragraph text and so do not stand out nor are in any other way attributed to the table that follows.  Before the change there was at least had a something that stood out to say what the stuff that followed was for the parameters used in this build (even if it had a bug where the parameters where rendered at the bottom of the page)

**Comment by fbonerandi** [comment]
https://github.com/jenkinsci/jenkins/pull/1987
New change to keep screen style

**Pull request closed without merging**